pkg/sql/sem/plpgsqltree/statements.go

```diff
@@ -115,6 +127,30 @@ func (s *PLpgSQLStmtIf) Format(ctx *tree.FmtCtx) {
 	ctx.WriteString("<NOT DONE YET>")
 }
 
+func (s *PLpgSQLStmtIf) walkStmt(visitor plpgsqlVisitor) {
+	visitor.Err = visitor.visit(s)
+
+	for _, thenStmt := range s.ThenBody {
+
+		visitor.Err = Walk(visitor, thenStmt)
```

Comment:
we probably don't need to call `Walk()` here, just `thenStmt.walkStmt(visitor)`. Same comment for others.